lux/core/frame.py

```diff
@@ -745,7 +745,7 @@ def current_vis_to_JSON(vlist, input_current_vis=""):
         current_vis_spec = {}
         numVC = len(vlist)  # number of visualizations in the vis list
         if numVC == 1:
```

Comment:
Is it possible here for the vlist to be of length 0?

Reply:
I don't think so, since we only call this when self.current_vis is not None (see line [729](https://github.com/lux-org/lux/pull/198/files#diff-b3f5355f16af7450f6bf0669466ac57ae020ed8c0a56105e8ece9c8c035a8879R729))

Reply:
Yup this was how it was done in the past as well